weave/scorers/utils.py

```diff
@@ -1,8 +1,55 @@
 import json
+import os
+from pathlib import Path
```

Comment:
we seem to be using a combination of `Path` and `os` dir.  Is there a reason why we can't just prefer `Path`?